ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/zones/multisubnets/external-lb/provision/cucushift-installer-rehearse-vsphere-ipi-zones-multisubnets-external-lb-provision-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-ipi-zones-multisubnets-external-lb-provision
+  steps:
+  - chain: ipi-conf-vsphere-zones-multisubnets-external-lb
+  - chain: ipi-install-vsphere
+  - ref: enable-qe-catalogsource
+  documentation: |-
```

Comment:
let's add the `cucushift-installer-check` chain to the provision-chain too

Reply:
oh, thanks for reminding.